.github/workflows/python_cibuildwheel.yml

```diff
@@ -0,0 +1,233 @@
+
+    - name: Setup Deps
+      shell: bash
+      run: |
+        if [ "$RUNNER_OS" == "Windows" ]; then
+          # C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
+          MSVC_DIR=$(cmd.exe /c "vswhere -products * -requires Microsoft.Component.MSBuild -property installationPath -latest")
+          echo "Latest is: $MSVC_DIR"
+          echo "MSVC_DIR=$MSVC_DIR" >> $GITHUB_ENV
+          # add folder containing vcvarsall.bat
+          echo "$MSVC_DIR\VC\Auxiliary\Build" >> $GITHUB_PATH
+        fi
+
+    - name: Build and test wheels
+      env:
+        COOLPROP_CMAKE: ${{ matrix.cmake_compiler}},${{ matrix.bitness }}
+        CIBW_ENVIRONMENT: COOLPROP_CMAKE=${{ matrix.cmake_compiler }},${{ matrix.bitness }}
+        MACOSX_DEPLOYMENT_TARGET: 10.9
+        CIBW_ENVIRONMENT_MACOS: MACOSX_DEPLOYMENT_TARGET=10.9 SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
+        CIBW_BEFORE_BUILD: pip install setuptools wheel Cython requests jinja2 pyyaml
+        CIBW_BEFORE_ALL_WINDOWS: call vcvarsall.bat ${{ matrix.arch }}
+        CIBW_BUILD: cp${{ matrix.python }}-${{ matrix.platform_id }}
+        CIBW_ARCHS: all
+        CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.manylinux_image }}
+        CIBW_MANYLINUX_I686_IMAGE: ${{ matrix.manylinux_image }}
+        CIBW_TEST_SKIP: "*-macosx_arm64"
+        # CIBW_TEST_COMMAND: python -c 'from CoolProp.CoolProp import get_global_param_string; print("CoolProp gitrevision:", get_global_param_string("gitrevision"))'
+        CIBW_BUILD_VERBOSITY: 1
+
+      run: |
+        python -m pip install cibuildwheel
+        python -m cibuildwheel --output-dir wheelhouse ./wrappers/Python
+
+    - name: Store artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        path: wheelhouse/*.whl
+
+  upload_python_bindings_to_pypi:
+    needs: python_bindings
+    if: always()
+    name: Upload to PyPi
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9.x
+
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine requests packaging
+        mkdir wheels
+
+    - name: Download ALL wheels
+      uses: actions/download-artifact@v2
+      with:
+        path: ./wheels
+
+    - name: Display structure of downloaded files
+      working-directory: ./wheels
+      run: |
+        ls -R
+
+    - name: Publish wheels to (Test)PyPI
+      # TODO: for now I'm effectively disabling uploading to testpypi on each build
+      if: contains(github.ref, 'refs/tags')
+      env:
+        TWINE_USERNAME: __token__
+      run: |
+
+        if [[ "$GITHUB_REF" == *"refs/tags"* ]]; then
+          TWINE_REPOSITORY=pypi
+          TWINE_PASSWORD=${{ secrets.PYPI_TOKEN }}
+        else
+          TWINE_REPOSITORY=testpypi
+          TWINE_PASSWORD=${{ secrets.TESTPYPI_TOKEN }}
```

Comment:
To enable upload to PyPi and TestPyPi, you would need to add a repo secrets **PYPI_TOKEN** / **TESTPYPI_TOKEN** set to the token you get there.

Reply:
Got it, I can do that when the time comes